contracts/cryptoblades.sol

```diff
@@ -144,28 +139,13 @@ contract CryptoBlades is Initializable, AccessControlUpgradeable {
     Blacksmith public blacksmith;
 
     struct MintPayment {
-        bytes32 blockHash;
         uint256 blockNumber;
         address nftAddress;
         uint count;
     }
 
     mapping(address => MintPayment) mintPayments;
-
-    struct MintPaymentSkillDeposited {
-        uint256 skillDepositedFromWallet;
-        uint256 skillDepositedFromRewards;
-        uint256 skillDepositedFromIgo;
-
-        uint256 skillRefundableFromWallet;
-        uint256 skillRefundableFromRewards;
-        uint256 skillRefundableFromIgo;
-
-        uint256 refundClaimableTimestamp;
-    }
-
-    uint256 public totalMintPaymentSkillRefundable;
-    mapping(address => MintPaymentSkillDeposited) mintPaymentSkillDepositeds;
```

Comment:
Removing these things will make the contract not backwards-compatible, so they must be left in.